openapi/openapi_client_test.go

```diff
@@ -1029,4 +1421,75 @@ func TestProviderClientDelete(t *testing.T) {
 			})
 		})
 	})
+
+	Convey("Given a providerClient set up with stub client that returns some response", t, func() {
+		httpClient := &http_goclient.HttpClientStub{}
+		providerClient := &ProviderClient{
+			openAPIBackendConfiguration: &specStubBackendConfiguration{
+				host:        "wwww.host.com",
+				basePath:    "/api",
+				httpSchemes: []string{"http"},
+			},
+			httpClient:            httpClient,
+			providerConfiguration: providerConfiguration{},
+			apiAuthenticator: &specStubAuthenticator{
+				authContext: &authContext{
+					headers: map[string]string{},
+				},
+			},
+		}
+		Convey("When providerClient DELETE  method is called with a SpecV2Resource that has a subresource path, a requestPayload, an empty responsePayload and the resource parentID", func() {
```

Comment:
What is "a requestPayload, an empty responsePayload" referring to?

Reply:
updated